CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 1.15.0
+- Update submodule to 3.5.1 [(#929)](https://github.com/algorand/indexer/pull/929)
```

Comment:
@algobarb These changelog notes point to _indexer_.  Feels like something is off here.  Double checking if it's the intended changeset?

Reply:
ah sorry, doing a couple releases at a time, copied the wrong ones in. We need to get this automated. Thanks for catching it!